insight/src/main/java/info/nightscout/androidaps/plugins/pump/insight/LocalInsightPlugin.java

```diff
@@ -1450,6 +1450,12 @@ private void processBolusDeliveredEvent(String serial, BolusDeliveredEvent event
                             bolusID.getId(),
                             PumpType.ACCU_CHEK_INSIGHT,
                             serial);
+            if (!isFakingTempsByExtendedBoluses())
```

Comment:
the above `syncExtendedBolusWithPumpId` doesn't need this?

Reply:
I thought, but I found a bug if I try to Cancel an extended bolus from button on Action Tab when fake tbr is false:
Extended Bolus is correctly cancelled on pump, but it remains active in overview, Action Tab and treatment tab...
I thought I miss this pumpSync call in driver (it works with this call it's ok), but it's possible that the bug is not in the driver but in pumpSync interface. (I didn't check with virtual pump...)

Reply:
Finally it works with Virtual Pump, So I have to understand what's wrong in insight driver (because it works with fake TBR...)